docs/compare-evidence-strings.md

```diff
@@ -0,0 +1,73 @@
+# Comparison of evidence strings
+
+When major updates to the pipeline are implemented, an important measure of control is comparing evidence strings before and after the update for the same input data. This protocol contains commands which help do this.
+
+## Install JQ locally
+JQ is a command line JSON processor.
+```bash
+wget -q -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+chmod a+x jq
+```
+
+## Sort keys in evidence strings
+This makes them more readable and helps comparison through word diff.
+```bash
+sort_keys () {
+  cat $1 \
+    | ./jq -S "." --tab \
+    | tr -d '\t\n' \
+    | sed -e 's|}{|}~{|g' \
+    | tr '~' '\n' \
+  > $2
+}
+export -f sort_keys
+
+sort_keys /nfs/production3/eva/opentargets/batch-2020-02/evidence_strings/evidence_strings.json old.json &
```

Comment:
Sorry to be a broken record but could we take out the server file paths and have a generic example since this is a public repo? If this example is important, I suggest to include it in the ticket.